apps/nextjs/src/app/[locale]/manage/tools/api/components/api-keys.tsx

```diff
@@ -46,6 +65,20 @@ export const ApiKeysManagement = ({ apiKeys }: ApiKeysManagementProps) => {
           </Group>
         ),
       },
+      {
+        header: t("table.header.actions"),
+        Cell: ({ row }) => (
+          <Group gap="xs">
+            <Button
+              onClick={() => handleDelete(row.original.id)}
+              leftSection={<IconTrashFilled size="1rem" />}
+              loading={isPendingDelete}
+            >
+              {t("button.deleteApiToken")}
+            </Button>
+          </Group>
+        ),
+      },
     ],
     [t],
```

Comment:
Add dependencies to array

Reply:
This results in a crash. Could you quickly look into this? I don't understand what's going on

Reply:
Do you mean this?
![image](https://github.com/user-attachments/assets/18a35695-c701-4d6b-8b0c-ea5876c8b727)

This also happens on dev for that page, so probably an issue with swagger-ui

Reply:
See https://github.com/swagger-api/swagger-ui/issues/10212

Reply:
Yes, I mean that. Should I add them to the array anyway?

Reply:
yeah I would say so

Reply:
I have done so. Now get "1 error" at the bottom of the page in development mode. Can we merge this safely without prod breaking?

Reply:
I would guess so as it also happend on dev